sdk/test-utils/test-utils/test/tracing/instrumentorTracingSpanTest.spec.ts

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Pretend this is a real client with real methods - like QueryLogsClient
```

Comment:
We can delete this comment. Also, is it possible to split this file up? 

TestTracingSpan describe block -> testTracingSpan.spec.ts
TestInstrumenter describe block -> testInstrumenter.spec.ts
etc?

Reply:
done